Parse fails on lists

## Describe the Bug

<!-- A clear and concise description of what the bug is. -->
YamlPeek fails when the yaml root is a list

## Steps to Reproduce

<!-- Tell us how to reproduce the issue. Ideally provide a failing unit test. -->
Feed a list like:

```yaml
- name: Star Trek
  stuff: yes
- name: Star Wars
  stuff: no
```

## Expected Behavior

parses correctly

## Exception with Stack Trace

<!-- If you see an exception, put the WHOLE THING here. -->

```text
"MyTarget.targets" (DoStuff target) (1) ->
(EnsureOutputFiles target) ->
  MyTarget.targets(31,5): error MSB4018: The "YamlPeek" task failed unexpectedly.
MyTarget.targets(31,5): error MSB4018: Newtonsoft.Json.JsonReaderException: Error reading JObject from JsonReader. Current JsonReader item is not an object: StartArray. Path '', line 1, position 1.
MyTarget.targets(31,5): error MSB4018:    at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader, JsonLoadSettings settings)
MyTarget.targets(31,5): error MSB4018:    at Newtonsoft.Json.Linq.JObject.Parse(String json, JsonLoadSettings settings)
MyTarget.targets(31,5): error MSB4018:    at YamlPeek.Execute() in /_/src/YamlPeek/YamlPeek.cs:line 67
MyTarget.targets(31,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Execute()
MyTarget.targets(31,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()
```

## Version Info

<!-- Main project version and other relevant dependencies you are using. -->
1.0.0

## Additional Info

<!-- Add any other context about the problem here. -->

<!-- sl -->
[![Back this issue](https://raw.githubusercontent.com/devlooped/SponsorLink/main/docs/assets/img/separator.png "Back this issue")](https://www.devlooped.com/SponsorLink/github/issues/?s=devlooped)
[![Back this issue](https://img.shields.io/badge/backed-%240-EA4AAA?logo=githubsponsors "Back this issue")](https://www.devlooped.com/SponsorLink/github/issues/?s=devlooped)
<!-- sl -->

--- a/src/Tests/Tests.cs
+++ b/src/Tests/Tests.cs
@@ -12,6 +12,23 @@ using Xunit.Abstractions;
 
 public record Tests(ITestOutputHelper Output)
 {
+    [Fact]
+    public void CanParseYamlList()
+    {
+        var task = new YamlPeek
+        {
+            Content =
+                """
+                - name: Star Trek
+                  stuff: yes
+                - name: Star Wars
+                  stuff: no
+                """,
+        };
+
+        Assert.True(task.Execute());
+    }
+
     [Theory]
     [MemberData(nameof(GetTargets))]
     public void Run(string file, string name, bool failure = false, string? code = null)

--- a/src/YamlPeek/YamlPeek.cs
+++ b/src/YamlPeek/YamlPeek.cs
@@ -65,7 +65,7 @@ public class YamlPeek : Task
         }).Deserialize(content);
 
         var json = JsonConvert.SerializeObject(data);
-        var jobj = JObject.Parse(json);
+        var jobj = JToken.Parse(json);
 
         Result = jobj.SelectTokens(Query)
             // NOTE: we cannot create items with empty ItemSpec, so skip them entirely.

